test/test_op_aliases.py

```diff
@@ -201,7 +194,7 @@ def _fn(t):
                 arg_string = ', '.join((str(arg) for arg in info.get_args(device)))
                 script = fn_template.format(alias_name=info.alias_name, args=arg_string)
             else:
-                is_input_tensor_list = isinstance(info.get_input(device), collections.Sequence)
+                is_input_tensor_list = isinstance(info.get_input(device), Sequence)
```

Comment:
I don't think you need to update this file, since this file doesn't currently use any OpInfos to test.

Reply:
`Sequence` here is imported from `collections.abc` to replace `collections.Sequence`.
The reason for the change is that `collections.Sequence` will be deprecated after Python 3.9.
```python
In [4]: collections.Sequence
/root/anaconda3/envs/xx/bin/ipython:1: DeprecationWarning: Using or importing the ABCs from 'collections' 
instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
```
Other changes for `test_op_aliases.py` has been reverted in the latest PR.